frontend/measure/src/app/details/details.component.ts

```diff
@@ -0,0 +1,172 @@
+        },
+        { name1: 'Amplitude:', amplitude: measurement.amplitude },
+        { name1: 'Angle frequency:', angle_freq: measurement.angle_freq },
+        { name1: 'Phase:', phase: measurement.phase },
+        { name1: 'Offset', offset: measurement.offset },
+      ];
+      this.measurementName = measurement.name;
+      this.measurement = measurement;
+    });
+  }
+
+  values: number[];
+  ids: number[];
+
+  getrhValues(): void {
+    this.backend.get_rh_values_for_measurement(this.id).subscribe((rhValue: PaginatedList<RhValue>) => {
+      this.values = rhValue.results.map(v => v.value);
+      this.ids = rhValue.results.map(i => i.id);
+      this.chartData = [
+        { data: this.values, label: 'Graph of measured Rh-values' },
+      ];
+      this.chartLabels = this.ids;
+      let datarhValue = rhValue.results.map(function (obj) {
+        return {
+          id: obj.id,
+          value: obj.value,
+        };
+      });
+      let dataLabels = [
+        { name: '', name2: '' },
+        { name: 'Measurments Id', name2: 'Rh Values' },
+      ];
+      this.measurementData = this.measurementData
+        .concat(dataLabels)
+        .concat(datarhValue);
+      this.rhValue = rhValue;
+    });
+  }
+
+  ngOnInit(): void {
+    this.getMeasurement(), this.getrhValues();
+  }
+
+  options = {
+    fieldSeparator: ';',
+    quoteStrings: '"',
+    decimalseparator: '.',
+    showLabels: true,
+    showTitle: true,
+    title: '',
+    useBom: false,
+    noDownload: false,
+    headers: [],
+  };
+
+  download_image() {
+    var canvas = document.getElementsByTagName('canvas')[0].toDataURL();
+    var link = document.createElement('a');
+    link.download = this.measurementName;
+    link.href = canvas;
+    link.click();
+  }
+
+  chartOptions = {
+    responsive: true,
+    scales: {
+      yAxes: [
+        {
+          scaleLabel: {
+            display: true,
+            labelString: 'Value of Rh',
+          },
+        },
+      ],
+      xAxes: [
+        {
+          scaleLabel: {
+            display: true,
+            labelString: 'Measurement ID',
+          },
+        },
+      ],
+    },
+  };
+
+  chartData = [{ data: [0], label: '' }];
+
+  chartLabels = [0];
+
+  lineChartColors: Color[] = [
+    {
+      // red
+      backgroundColor: 'rgba(0,0,0,0)',
+      borderColor: 'red',
+      pointBackgroundColor: 'rgba(148,159,177,1)',
+      pointBorderColor: '#fff',
+      pointHoverBackgroundColor: '#fff',
+      pointHoverBorderColor: 'rgba(148,159,177,0.8)',
+    },
+    {
+      // dark grey
+      backgroundColor: 'rgba(0,0,0,0)',
+      borderColor: 'rgba(77,83,96,1)',
+      pointBackgroundColor: 'rgba(77,83,96,1)',
+      pointBorderColor: '#fff',
+      pointHoverBackgroundColor: '#fff',
+      pointHoverBorderColor: 'rgba(77,83,96,1)',
+    },
+  ];
+
+  onChartClick(event) {
+    console.log(event);
+  }
+
+  results() {
+    new ngxCsv(this.measurementData, this.measurementName, this.options);
```

Comment:
gör detta något?

Reply:
Lite osäker på vad komentaren syftar på.. 
Syftar du på new ngxCsv(this.measurementData...) så är det den funktioner som exporterar datan till csv fil. 
onChartClick(event) tror jag inte gör nått så tog bort den.